### PR TITLE
Changing verlock behavior to allow new greater than or equal behavior

### PR DESCRIPTION
**Pull request by ShelbyZ**
- Rework verlock to an integer
- Add new state allowing greater than or equal matching
- Update console input to allow toggling between states
- Update handling of new verlock

### CODE REVIEW

**Comment by ShelbyZ** [comment]
@KnowOne134 - this might be close to inline with what you were suggesting

**Comment by TeoTwawki** [comment]
This 3rd option was the original behavior when I created it, which @teschnei killed because people kept reporting "bugs" from their version being old.

**Comment by TeoTwawki** [comment]
Also I assume changing bools into numbers and then using the numbers as a bool isn't wanted, since that's what this was and it got changed from that..We do that all over the core I don't know why \\\_(o.O)\_/

**Comment by KnowOne134** [comment]
Yes thats what i was exactly looking for. We do keep the server updated amd version locked but there is a few week transition before I find the time to do the updates ussually. So something that lets me lock out the lower ver but let in what it was and let in the new players also would be very helpfull. And give my current players time when i announve uts time to update to newest version

File: src/login/login.cpp [hunk 1]
```diff
@@ -131,8 +131,23 @@ int32 do_init(int32 argc, char** argv)
...
+                        switch (version_info.ver_lock)
+                        {
+                            case 0:
+                                value = "disabled";
+                                break;
+                            case 1:
+                                value = "enabled - strict";
+                                break;
+                            case 2:
+                                value = "enabled - greather than or equal";
```
Comment (takhlaq) [inline src/login/login.cpp]: that's pretty greath

**Pull request merged**